### PR TITLE
Add New Profile: lunc_nymph

### PR DESCRIPTION
**Pull request by luncnymph**
Hi, I'd like to add a new validator profile:

Moniker: lunc_nymph

Twitter: @lunc_nymph

Operator address
terravaloper1qrgwphlf9c64m5ys6pzsvy055ud03e4kr7sq8u

Account address
terra1qrgwphlf9c64m5ys6pzsvy055ud03e4kr3uah0

Thank you!

### CODE REVIEW

**Review by terran6 — CHANGES_REQUESTED** [review]
Please ensure that your `profile.json` file is in JSON format.  Also, it seems that you have a lot of information in your README.  I would advise taking a look at other validators READMEs as well as their `profile.json` files to get an idea of important information to include.  Thank you!

**Comment by luncnymph** [comment]
Will make the change. Thanks

**Comment by luncnymph** [comment]
changes made. please review and let me know if it works. Thanks a lot!

**Review by terran6 — APPROVED** [review]
✨

File: validators/terravaloper1qrgwphlf9c64m5ys6pzsvy055ud03e4kr7sq8u/profile.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "contact": {
+    "email": "lunc_nymph@proton.me",
+    "keybase": "https://keybase.io/lunc_nymph",
+    "twitter": "https://twitter.com/lunc_nymph",
```
Comment (terran6) [inline validators/terravaloper1qrgwphlf9c64m5ys6pzsvy055ud03e4kr7sq8u/profile.json:5]: Trailing commas are not allowed on JSON.  If you remove this, you should be ready to go!

Comment (luncnymph) [reply validators/terravaloper1qrgwphlf9c64m5ys6pzsvy055ud03e4kr7sq8u/profile.json]: sorry, changed. thanks

**Review by luncnymph — COMMENTED** [review]
removed. please review. thanks

**Review by terran6 — COMMENTED** [review]
✨✨

**Review by terran6 — APPROVED** [review]
✨✨✨

**Comment by luncnymph** [comment]
When can I see the blue check on Station? Thanks

**Pull request merged**